### ISSUE TITLE
Move to Tentacoli as client / server transport over Nes

### ISSUE THREAD

**Issue by mcdonnelldean** [vidi-insights/vidi-dashboard#123]
Tentacoli now supports browser and will enable live and historic requests that are pattern matched. We will move to this over nes.

https://github.com/mcollina/tentacoli

cc/ @mcollina

**Comment by mcollina** [comment]
Just to clarify everybody else, it's probably worth explaining why.
Il giorno mer 10 feb 2016 alle 22:59 Dean McDonnell <
notifications@github.com> ha scritto:

**Comment by mcdonnelldean** [comment]
@mcollina Good point, I thought, because it is awesome would do :D

Will edit above.

**Comment by mcdonnelldean** [comment]
@mcollina Question on this, based on the first cut I wrote I need some way to Id a client. Is this something Tentacoli can do or am I best just sending a nid from the client in the request?

**Comment by mcollina** [comment]
Each request in Tentacoli gets a uuid. Should we look at ways to surface this? https://github.com/mcollina/tentacoli/blob/master/tentacoli.js#L238

**Comment by mcdonnelldean** [comment]
@mcollina I think I need per client. A client might have multiple subscriptions running.

**Comment by mcollina** [comment]
I think you will have to do it on top of tentacoli, as par of the websocket-stream/hapi/authentication stuff.

**Comment by mcdonnelldean** [comment]
@mcollina Agreed, I have to write a client-side wrap anyway to add context stuff so it makes sense to do it there

**Comment by mcollina** [comment]
I will put all of it in an hapi plugin, similar to nes.

**Comment by mcdonnelldean** [comment]
@mcollina Great idea, I will make this it's own issue on Tentacoli
